website/docs/reference/rbac.md

```diff
@@ -3,37 +3,45 @@ id: rbac
 title: Role-based Access control
 ---
 
-This document forms the specifications for [Role-Based Access Control](https://en.wikipedia.org/wiki/Role-based_access_control) which was introduced as part of the **Unleash v4 release**.
+This document forms the specifications
+for [Role-Based Access Control](https://en.wikipedia.org/wiki/Role-based_access_control) which was introduced as part of
```

Comment:
I think it's usually nice to be explicit when we introduce an acronym (because it's used below)
```suggestion
for [Role-Based Access Control](https://en.wikipedia.org/wiki/Role-based_access_control) (RBAC) which was introduced as part of
```

Reply:
I'll add this, but it has nothing to do with my PR. I see the autoformatting here has touched more than it should have.